specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -0,0 +1,1426 @@
+          "x-nullable": false
+        },
+        "valueInteger": {
+          "description": "Integer value.",
+          "type": "integer",
+          "x-nullable": false
+        },
+        "valueArray": {
+          "description": "Array of field values.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FieldValue"
+          }
+        },
+        "valueObject": {
+          "description": "Dictionary of named field values.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/FieldValue"
+          }
+        },
+        "text": {
+          "description": "Extracted text content of the recognized field.",
+          "type": "string"
+        },
+        "boundingBox": {
+          "description": "Bounding box of the field text, if appropriate.",
+          "$ref": "#/definitions/BoundingBox"
+        },
+        "confidence": {
+          "description": "Qualitative confidence measure.",
+          "$ref": "#/definitions/Confidence"
+        },
+        "elements": {
+          "description": "List element references.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ElementReference"
+          }
+        }
+      }
+    },
+    "FieldValueType": {
+      "type": "string",
+      "description": "Type of field value.",
+      "enum": [
+        "string",
+        "date",
+        "time",
+        "phoneNumber",
+        "number",
+        "integer",
+        "array",
+        "object"
+      ],
+      "x-ms-enum": {
+        "name": "FieldValueType",
+        "modelAsString": false
+      },
+      "x-nullable": false
+    },    
+    "Confidence": {
+      "description": "Confidence value.",
+      "type": "number",
+      "minimum": 0,
+      "maximum": 1
+    },
+    "CopyRequest": {
+      "description": "Contract to initiate a copy request.",
+      "type": "object",
+      "properties": {
+        "targetInformation": {
+          "description": "Information about target subscription.",
+          "type": "object",
+          "properties":{
+            "endpoint": {
+              "description": "Get or set enpoint path.",
+              "maxLength": 2048,
+              "minLength": 0,
+              "type": "string"
+            },
+            "resourceId": {
+              "description": "Get or set resource identifier.",
+              "maxLength": 2048,
+              "minLength": 0,
+              "type": "string"
+            }
+          }
+        },
+        "copyConfig": {
+          "description": "Copy operation options.",
+          "type": "object",
+          "required":[
+            "deleteSource"
+          ],
+          "properties":{
+            "deleteSource": {
+              "description": "Indicate deletion of source model artifacts.",
+              "type":"boolean",
+              "default":false
+            },
+            "supportCrossTenetCopy": {
```

Comment:
```suggestion
            "supportCrossTenantCopy": {
```

Reply:
This won't get flagged by the spellcheck, but i don't think it's the right word https://en.wikipedia.org/wiki/Tenet

Reply:
Agreed. Fixed.